sys/multi_content_bucket.js

```diff
@@ -378,7 +378,11 @@ class MultiContentBucket {
                         });
                     })
                     .catch({ status: 404 }, () => {
-                        // Ignore the 404 if we don't have the timeline.
+                        // Log the 404 if we don't have the timeline.
+                        hyper.log('error/noindex', {
+                            msg: 'Empty revision timeline',
+                            page_title: rp.key
```

Comment:
Do we need the domain logged as well, or will that be available elsewhere?

Reply:
The domain should be added by default.